frontend/kesaseteli/employer/src/pages/login.tsx

```diff
@@ -1,25 +1,52 @@
+import { IconSignin } from 'hds-react';
+import useLogin from 'kesaseteli/employer/hooks/backend/useLogin';
 import { GetStaticProps, NextPage } from 'next';
 import { useRouter } from 'next/router';
+import { useTranslation } from 'next-i18next';
 import * as React from 'react';
 import Container from 'shared/components/container/Container';
 import withoutAuth from 'shared/components/hocs/withoutAuth';
 import Layout from 'shared/components/Layout';
 import useClearQueryParams from 'shared/hooks/useClearQueryParams';
 import getServerSideTranslations from 'shared/i18n/get-server-side-translations';
 
+import { $Notification, $PrimaryButton } from '../components/application/login.sc';
+
 const Login: NextPage = () => {
   useClearQueryParams();
+  const { t } = useTranslation();
   const {
     query: { logout, error, sessionExpired },
   } = useRouter();
+  const login = useLogin();
+
+  let notificationLabel = t(`common:loginPage.infoLabel`)
+  if (logout) {
+    notificationLabel = t(`common:loginPage.logoutLabel`)
+  } else if (error) {
+    notificationLabel = t(`common:loginPage.errorLabel`)
+  } else if (sessionExpired) {
+    notificationLabel = t(`common:loginPage.sessionExpiredLabel`)
+  }
+  const notificationContent = !logout && !error && !sessionExpired && t(`common:loginPage.infoContent`)
+  const notificationType = error || sessionExpired ? "error" : "info"
+
   return (
     <Container>
-      <Layout headingText="Työnantajan liittymä">
-        {logout && <p>Olet kirjautunut ulos</p>}
-        {error && <p>Tapahtui tuntematon virhe. Kirjaudu uudelleen sisään.</p>}
-        {sessionExpired && (
-          <p>Käyttäjäsessio vanhentui. Kirjaudu uudelleen sisään.</p>
-        )}
+      <Layout headingText="">
```

Comment:
empty values are discouraged, perhaps you could change the Layout component so that it allows headingText to be optional